clients/python/play.py

```diff
@@ -1,5 +1,38 @@
+import json
+import sys
+
+from game.api import API
+from game.models import Game
+
+
 def main():
-    print("TODO")
+    team_secret = fetch_team_secret()
```

Comment:
I prefer the distinction, but if you guys all prefer `player` we can change it as part of a different PR (to update the API as well).